apps/mobile-wallet/src/features/localization/useLocalization.ts

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import 'dayjs/locale/fr'
+import 'dayjs/locale/el'
+
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import updateLocale from 'dayjs/plugin/updateLocale'
+import { useEffect } from 'react'
+import { getLocales } from 'react-native-localize'
+
+import { sendAnalytics } from '~/analytics'
+import i18next from '~/features/localization/i18n'
+import { Language, languageOptions } from '~/features/localization/languages'
+import { systemLanguageMatchFailed, systemLanguageMatchSucceeded } from '~/features/localization/localizationActions'
+import { shortRelativeTime } from '~/features/localization/relativeTime'
+import { useAppDispatch, useAppSelector } from '~/hooks/redux'
+
+dayjs.extend(updateLocale)
+dayjs.extend(relativeTime)
+
+export const useLocalization = () => {
```

Comment:
Nit: what about calling this `useInitializeLocalization` for clarity sake, following what we've done already with `useInitializeClient`?

Reply:
I would, but it also takes care of changing the language of i18n when the language in the settings changes, so it's not only about initializing.